spec/connect/hwinfo/base_spec.rb

```diff
@@ -39,10 +39,18 @@
     end
 
     context 'on AWS hypervisors' do
-      let(:dmidecode_output) { File.read(File.join(fixtures_dir, 'dmidecode_aws.txt')) }
+      context 'on regular instances' do
+        let(:dmidecode_output) { File.read(File.join(fixtures_dir, 'dmidecode_aws.txt')) }
+        it 'detects it' do
```

Comment:
Would be `it 'detects the provider'` a bit more descriptive?

Reply:
Sure, done.